src/ORM/Table.php

```diff
@@ -352,6 +355,8 @@ public function setTable($table)
     /**
      * Returns the database table name.
      *
+     * This can include the database schema name if set using setTable.
```

Comment:
```suggestion
     * This can include the database schema name if set using `setTable()`.
```

Reply:
Is there a way to verify formatting in comments shows up correctly?

Reply:
None other than generating the api docs yourself locally using https://github.com/cakephp/cakephp-api-docs.

Else you can just compare existing docblocks and the generated api docs at api.cakephp.org.